pkg/controllers/management/etcdbackup/etcdbackup.go

```diff
@@ -162,6 +162,21 @@ func (c *Controller) clusterBackupSync(ctx context.Context, interval time.Durati
 	return nil
 }
 
+type FilterFunc = func(backup *v3.EtcdBackup) bool
+
+func filterBackups(backups []*v3.EtcdBackup, filter ...FilterFunc) []*v3.EtcdBackup {
```

Comment:
A comment regarding how the filter functions are handled (or-ed versus and-ed) for each backup would be helpful.